widgetry/src/widgets/spinner.rs

```diff
@@ -27,11 +28,23 @@ pub struct Spinner {
 }
 
 impl Spinner {
-    pub fn widget(ctx: &EventCtx, (low, high): (isize, isize), current: isize) -> Widget {
-        Widget::new(Box::new(Self::new(ctx, (low, high), current)))
+    pub fn widget<S: Into<String>>(
+        ctx: &EventCtx,
+        label: S,
```

Comment:
Just a take-it-or-leave-it-code-style nit, I prefer the `label: impl Into<String>` so you don't have to backtrack to see what `S` is. Is there a reason I shouldn't?

Reply:
I also like this style better; I didn't know it could work like this to specify an existential type. I'll switch more of the codebase over to this (and also try the `AsRef<str>` idea) as warmup.

Reply:
I've attempted, and failed, several times to really understand what exactly an existential type is. I think I might just not care enough. 🤣